.github/workflows/test.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       run: |
         set -x
         pushd go-controller
-        make check
+           RACE=1 make check
```

Comment:
Why not leave this as `make check` and make the default behaviour in `test-go.sh` be to enable the race detector. This way we won't forget to run with RACE=1 locally and be surprised by failures in CI

Reply:
You are right. I thought about this yesterday, but got caught up in other things. I will modify that